### DISCUSSION TITLE
Midi input circuit support

### DISCUSSION THREAD

**Discussion by lcollins1989** [Builty/TonexOneController#4] [Ideas]
First of all. Outstanding work on this project! I was about to embark on similar but you’ve beaten me to it!

Quick question as I’m more familiar with Arduino/C++ style programming (self taught) so I'm not certain how to add it myself. 

How hard would it be to add midi control via a midi input circuit? I'd love to be able to control your setup from another device directly.

**Comment by Builty** [comment]
Thank you.
Midi over serial should not be hard to do. 
I'll add it to the issue list.

**Reply by Builty** [reply]
I've added support, not curently able to test it

**Reply by lcollins1989** [reply]
Wow that was fast!

My waveshare controller should arrive today so I will try and text as soon as I can.
